### DISCUSSION TITLE
Does Flare solver do captchas?

### DISCUSSION THREAD

**Discussion by ArminEskandarii** [FlareSolverr/FlareSolverr#1377] [Q&A]
Hi, I didn't understand from the read me file.

Does Flaresolver do cloudflare captchas? (The ones that require just one click)

I also see that it says, Some captchas do show a question thing, But that's not what I'm talking about, The ones that only require click.

Does it solve these kind of captchas on the default Docker deploy method?

Or should I activate it myself somehow?

**Comment by ilike2burnthing** [comment]
Yes, although see pinned issues and recent PRs.
